src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      'x': 0, 'y': 0,
+    },
+    get info() {
+      return (
+        `name: ${
+          this.name}, chip version: ${this.version}, wheels: ${this.wheels}`
+      );
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.y = this.coords.y + steps;
```

Comment:
```suggestion
      if (steps > 0) {
        this.coords.y += steps;
      }
```
You will return this anyway. Just check if steps > 0.
Also, you can join arithmetic operators with assignment operators to make your code cleaner
Fix all cases.